apps/authentication/views.py

```diff
@@ -83,9 +85,12 @@ def register(request):
 
                 # Create the registration token
                 token = uuid.uuid4().hex
-                rt = RegisterToken(user=user, email=email.email, token=token)
-                rt.save()
-
+                try:
+                    rt = RegisterToken(user=user, email=email.email, token=token)
+                    rt.save()
+                    log.info('Successfully registered token for %s' % request.user)
+                except:
+                    log.error('Failed to register token for %s' % request.user)
```

Comment:
I believe @myth was the one to puth the try-except block in there. After making token a unique field in the model it would probably fail if the token has been used before i guess(?).